src/tapis-redux/src/streams/projects/useProjects.ts

```diff
@@ -0,0 +1,21 @@
+import { useSelector } from 'react-redux';
+import { list } from './actions';
+import { TapisState } from '../../store/rootReducer';
+import { ProjectsListCallback } from './types';
+import * as Streams from "@tapis/tapis-typescript-streams";
+import { Config } from 'tapis-redux/types';
+
+export interface ProjectsListParams {
+  onList?: ProjectsListCallback,
+  request?: Streams.ListProjectsRequest
+}
+
+const useProjects = (config?: Config) => {
```

Comment:
There should be one top level hook for useStreams, which aggregates most of the features here into subobjects. so useStreams could return { projects, sites, instruments } where each is an object that contains the associated functionality with those sub-services.